js_modules/dagit/packages/eslint-config/rules/index.js

```diff
@@ -1,19 +0,0 @@
-/* eslint-disable */
-const projectName = 'dagster-rules';
-
-const rules = {
-  'missing-graphql-variables-type': require('./missing-graphql-variables-type').rule,
-};
-
-module.exports = {
```

Comment:
Maybe we should keep this stub here so it's easy to add rules in the future? For example we could lint routes that don't use the analytics tracking APIs.

Reply:
Sure, that's reasonable